mock/api.js

```diff
@@ -281,21 +287,59 @@ export const mockSession = {
   },
 };
 
+export const mockEndpoints = {
+  api: {
+    controllers_list: '/api/v1/controllers/list',
+    controllers_months: '/api/v1/controllers/months',
+    datasets_list: '/api/v1/datasets/list',
+    datasets_detail: '/api/v1/datasets/detail',
+
+    // NOTE: these are not implemented in the server yet, but are defined
+    // below as mocks for testing
+    datasets_toc: '/api/v1/datasets/toc',
+    datasets_samples: '/api/v1/datasets/samples',
+    datasets_timeseries: '/api/v1/datasets/timeseries',
+    mappings: '/api/v1/index/mappings',
```

Comment:
I guess it should be `/api/v1/index/mappings/` since that how its implemented in server now

Reply:
You should add that when you change the dashboard to actually use it. 😁